### ISSUE TITLE
Bladebound doesn't have a Black Blade?

### ISSUE THREAD

**Issue by Canaanchaos** [Vek17/TabletopTweaks-Base#460]
So, I just created a new character with the Bladebound Magus Archetype. And he just leveled to 3, which means I SHOULD get a black blade, right? I selected what form I want it to take and... nothing. Maybe I'm missing something, but there was no longsword that appeared equipped to my character, nor in my inventory, nor was there any active ability to summon the thing present on that character. Additionally, I used the Toybox to ADD a longsword called "Black Blade" but then it wouldn't let that character (or any) equip it. Am I missing something?

**Comment by Vek17** [comment]
Was this a respec or fresh character?

**Comment by Canaanchaos** [comment]
It was a fresh character.

**Comment by Edalwulf** [comment]
I've just gone through to level 3 myself, and found that I do receive my black blade. However, I've also found that the weapon enhancement abilities that the Magus and Spirit Hunter don't increase the enhancement properly. Instead, they immediately turn it to a +5 weapon, ignoring their capacity to to add abilities to weapons.

**Comment by Vek17** [comment]
That would not be related to the black blade, not sure what would be going on there would need to look into it. I haven't updated that system at all.

**Comment by Edalwulf** [comment]
I know that I don't have anything on in Toybox that _should_ effect that, but it's possible that there's some conflict there that isn't obvious.

**Comment by Canaanchaos** [comment]
I mean, if there's a way to manually add it, that would work for me. I did a party composed entirely of various fantasy characters and the second I saw the bladebound, I knew I needed to add Elric to the mix. Do you think it might be that, in two separate cases that I tried, it was adding a mercenary AFTER my characters were all level 3? Because it meant that, after level 1, levels 2 and 3 we're added at the same time. I'm just spitballing here. Haven't had a chance to try anything else out.

**Comment by Vek17** [comment]
Shouldn't matter, i know this was an issue in an older version of TTT-Core,
is that fully up to date?

**Comment by Canaanchaos** [comment]
It is. 0.3.2. I just created a character on a new game and used Toybox to level him up one level at a time. For me, at least, the problem does not occur if I level up one at a time. So from a new game, it's not an issue, but if I hire a mercenary at any level higher than 2, and I have to add multiple levels at once, that's where I wind up with issues.

**Comment by Christoph-Wagner** [comment]
I’m currently having this issue in the Midnight Isles DLC, no Black Blade, everything is up-to-date. I used multiple archetypes, and it’s Hexcrafter/Bladebound, but the more relevant part from the discussion seems to be that she was a merc hired for level 7. With the mode being last azata-lite, I have no save before I recruited her, is there a way to edit the save file to add the blade in while Toybox is not updated?
